project/scanners/components/ScannerSelect.vue

```diff
@@ -1,53 +1,110 @@
 <template>
-  <v-select
-    v-model="selectedScanner"
-    :items="scanners"
-    :item-props="getScannerDetials"
-    hide-details
-    variant="outlined"
-    item-title="endpoint"
-    density="compact"
-    clearable
-    return-object
-    label="Select a scanner"
-    placeholder="Select or create a scanner to run the scan on"
-    prepend-icon="mdi-shield-search"
-  >
-    <template #selection="{ item }">
-      <p
-        v-if="(item.raw.name || '').trim() !== ''"
-        class="mb-0"
+  <div>
```

Comment:
Not the correct UI. You should have a drop down. And in the top of the drop down a button add scan. Click on it and it shows the modal.